README.md

```diff
@@ -1,15 +0,0 @@
-# SendGrid
-
-Informatics 117 Team 6
-
-- Brenda La
-- David Pham
-- Eric Chou
-- Jose Gomez
-- Sheila Truong
-
-
-#### Files:
-
-- client.go: Generates random user ID and emails and sends to localhost:8081/retrieve
```

Comment:
These two files are in the same directory and the same `package main`. This can be confusing. For example, while I was able to `go run` each of them in the terminal, my code inspection complains that `main` is defined twice. Consider reorganizing by creating two separate folders `server` and `client`.  Also you can check out https://talks.golang.org/2014/organizeio.slide